site/assets/keedrop.js

```diff
@@ -0,0 +1,286 @@
+"use strict";
+
+function defaultAdapter(baseUrl) {
+  function request(url, data) {
+    var req = new XMLHttpRequest();
+    req.open(data ? "POST" : "GET", "https://cors-anywhere.herokuapp.com/" + url);
```

Comment:
I'll better add some proper CORS then? :sweat_smile:

Reply:
haha, no, I only put that there for local testing.
If you want to restrict the API to the keedrop server, then no CORS required.